### PR TITLE
Shatabdi/feature/send mail

### PR DESCRIPTION
**Pull request by sbswarna**

### CODE REVIEW

File: src/modules/core/mail-server/mailServer.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const path = require("path");
```
Comment (UitsHabib) [inline src/modules/core/mail-server/mailServer.js]: in src/modules/core -> create a folder email-service -> under email-service folder create a file email.service.js -> and keep the codes of this file.

File: src/modules/core/mail-server/mailServer.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const handlebars = require("nodemailer-express-handlebars");
+const nodemailer = require("nodemailer");
+
+require("dotenv").config();
+const { GMAIL_ADDRESS, GMAIL_PASSWORD } = process.env;
+
+function sendMail(receiver) {
+	const transporter = nodemailer.createTransport({
```
Comment (UitsHabib) [inline src/modules/core/mail-server/mailServer.js]: keep the transporter out of the sendMail function. As it is independent logic. so keep this in global scope

**Review by UitsHabib — CHANGES_REQUESTED** [review]

File: src/modules/core/mail-server/mailServer.js (hunk 2 above)
Comment (sbswarna) [reply src/modules/core/mail-server/mailServer.js]: Updated

File: src/modules/core/mail-server/mailServer.js (hunk 1 above)
Comment (sbswarna) [reply src/modules/core/mail-server/mailServer.js]: Updated

**Review by UitsHabib — APPROVED** [review]

**Pull request merged**